changelogs/fragments/1085-s3_lifecycle-reassure-that-configuration-is-complete-before-returning.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - s3_lifecycle: reassure that configuration is complete before returning
```

Comment:
```
2022-04-20 14:37:16.435242 | controller | ERROR: Found 1 changelog issue(s) which need to be resolved:
2022-04-20 14:37:16.435442 | controller | ERROR: changelogs/fragments/1085-s3_lifecycle-reassure-that-configuration-is-complete-before-returning.yml:0:0: section "bugfixes" list items must be type str not dict
```

```suggestion
  - s3_lifecycle - check that configuration is complete before returning (https://github.com/ansible-collections/community.aws/pull/1085).
```

Reply:
Hi, thanks for feedback. Change applied.